src/config/locales.json

```diff
@@ -8,6 +8,7 @@
 		"it": "Italiano",
 		"pt-br": "Português (Brasil)",
 		"ru": "Русский",
+        "zh-cn": "简体中文",
```

Comment:
Please use tabs here instead of spaces

Reply:
It seems that vscode replaces my tabs with spaces...To be honest, I didn't notice the difference between them, but I will pay attention in the future.

Reply:
Fixed.